backend/spec/requests/api/v1/users_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Sessions' do
+  describe 'PUT /me' do
+    let(:params) do
+      { user:, birthdate: }
+    end
+    let(:password_confirmation) { password }
+    let(:birthdate) { nil }
+    let(:user) { create(:user) }
+
+    before { user }
```

Comment:
You could use `let!` here instead of `before` ([docs](https://www.betterspecs.org/#let))